src/main/java/net/neoforged/moddevgradle/internal/ModDevPlugin.java

```diff
@@ -248,14 +256,26 @@ public void apply(Project project) {
             });
         });
 
+        configurations.create(CONFIGURATION_COMPILE_DEPENDENCIES, config -> {
```

Comment:
Why a second set of dependencies? Is there any difference with the runtime one besides the harmless extra jar?

Reply:
OK there is also `DevLaunch`. I think that's fine then.